benchmark/micro/compression/chimp/chimp_store.benchmark

```diff
@@ -0,0 +1,19 @@
+# name: benchmark/micro/compression/chimp/chimp_store.benchmark
+# description: Scanning a large amount of doubles
+# group: [chimp]
+
+name Chimp Insert
+group chimp
+storage persistent
+require_reinit
+
+load
+PRAGMA force_compression='uncompressed';
+DROP TABLE IF EXISTS temperatures;
+CREATE TABLE temperatures (
+	temperature DOUBLE
+);
+
+run
+INSERT INTO temperatures SELECT temp FROM 'benchmark/trainbenchmark/float_datasets/city_temperature.csv' t(temp), range(28);
```

Comment:
Yea since these datasets are quite large, I don't think we want to include them in the repo right?

I was wondering how we want to handle this, because on the one hand verifying that the compression ratio lives up to the results of the paper is important and we want to verify this, but on the other hand these giant datasets bloat the size of the repository by a lot.